app/src/main/java/com/brainwallet/ui/screens/home/composable/HomeSettingDrawerSheet.kt

```diff
@@ -15,25 +15,40 @@ import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.ModalDrawerSheet
```

Comment:
WIP: this is working @andhikayuana but I would prefer a more elegant view/screen refresh